loops-and-arrays.js

```diff
@@ -6,14 +6,17 @@
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
+console.log(names);
+
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
+for (let i = 0; i < names.length; i++) {
+    console.log(names [i] + "je");
```

Comment:
hier wordt nu alleen de "je" in de log geplaatst.
De opdracht vraagt om de array aan te passen.